.gitignore

```diff
@@ -10,6 +10,10 @@ awsconfig
 /public/views/index.html
 /public/views/error.html
 /emails/dist
+
+# Enterprise emails
+/emails/templates/for-enterprise-*
```

Comment:
Think a folder named enterprise that we can symlink in where all enterprise emails would be preferable but not sure if there is some problem with that. 

The emails get built and checked in currently in this folder, so we would need an enterprise folder ignored (and symlinked) here as well:

https://github.com/grafana/grafana/tree/master/public/emails

If a folder is tricky for some reason then a prefix like `enterprise_template-name.html` is best the other files already use underscore.

Reply:
> Think a folder named enterprise that we can symlink in where all enterprise emails would be preferable but not sure if there is some problem with that.

Yeah, I was thinking the same thing! But I didn't want to deal with refactoring of the grunt config for it, was planning to create an issue for that

Reply:
> The emails get built and checked in currently in this folder, so we would need an enterprise folder ignored (and symlinked) here as well:

> https://github.com/grafana/grafana/tree/master/public/emails

> If a folder is tricky for some reason then a prefix like enterprise_template-name.html is best the other files already use underscore.

Done!